### DISCUSSION TITLE
Change camera angle with button

### DISCUSSION THREAD

**Discussion by eyeSphere** [google/model-viewer#2039] [Q&A]
I'm trying to implement a button that changes the camera angle on click, but without success.

Should something like this be working? 

`function ButtonClick() {
modelViewer.cameraOrbit = '1deg 2deg 3m';
}
`

**Comment by 1kaiser** [comment]
`<model-viewer camera-controls camera-orbit="calc(-1.5rad + env(window-scroll-y) * 4rad) calc(0deg + env(window-scroll-y) * 180deg) calc(3m - env(window-scroll-y) * 1.5m)" src="../../shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut"></model-viewer>` try these. [https://modelviewer.dev/examples/stagingandcameras/](https://modelviewer.dev/examples/stagingandcameras/)

**Comment by elalish** [comment]
Yes, it should. Of course every time you click the button it'll take you back to the same place. Have you checked that `modelViewer != null`?
